### PR TITLE
Updates for hub listing; inclusion of extension icon

### PR DESCRIPTION
**Pull request by zyoutz-r7**
- Add extension.png for icon
- Update spec with format changes
- Slight updates to help.md (eg removal of Dev section)
NOTE: not necessary to rebuild plugin after merging; just for storing of files with project

### CODE REVIEW

File: help.md [hunk 1]
```diff
@@ -22,15 +22,13 @@ Additionally the plugin can be installed by manually building the hpi file and u
...
-### Development
+Additional documentation for installing and configuring the Jenkins plugin can be found on the [Rapid7 help pages](https://insightappsec.help.rapid7.com/docs/jenkins-integration).
 
-To run the plugin locally, cd to the root directory and invoke:
+### Installation
 
-```
-mvn hpi:run
-```
+The plugin may be installed using the plugin manager. For more information, see https://plugins.jenkins.io/insightappsec.
```
Comment (r-powell-r7) [inline help.md]: Dead link, need to remove the `.`

File: help.md [hunk 2]
```diff
@@ -22,15 +22,13 @@ Additionally the plugin can be installed by manually building the hpi file and u
 
 ## Technical Details
 
-### Development
```
Comment (r-powell-r7) [inline help.md]: Why remove this section? It's there for plugin developers (both without rapid7 and the community)

File: help.md [hunk 3]
```diff
@@ -22,15 +22,13 @@ Additionally the plugin can be installed by manually building the hpi file and u
...
+Additional documentation for installing and configuring the Jenkins plugin can be found on the [Rapid7 help pages](https://insightappsec.help.rapid7.com/docs/jenkins-integration).
 
-To run the plugin locally, cd to the root directory and invoke:
+### Installation
```
Comment (r-powell-r7) [inline help.md]: There is an installation section at the top of the readme already?

Comment (zyoutz-r7) [reply help.md]: Under `Contributions` of the help.md (used by the extension listing library) we mention referencing the README/repo.  Trying to keep developer specific content from the actual listing as it is intended for end users.

File: help.md (hunk 2 above)
Comment (zyoutz-r7) [reply help.md]: Under Contributions of the help.md (used by the extension listing library) we mention referencing the README/repo. Trying to keep developer specific content from the actual listing as it is intended for end users.

Comment (r-powell-r7) [reply help.md]: Gotcha, the development section in help file has us covered

File: help.md (hunk 1 above)
Comment (r-powell-r7) [inline help.md]:
```suggestion
The plugin may be installed using the plugin manager. For more information, see https://plugins.jenkins.io/insightappsec
```

**Pull request merged**